internal/commands/terraform/terraform.go

```diff
@@ -20,6 +20,9 @@ type TerraformOptions struct {
 
 var terraformLongDesc = templates.LongDesc(`
 	Run terraform command via terraform docker container.
+	By default terraform run in docker container.
+
+	At the same time, terraform will be downloaded and launched from ~/.ize/.terraform.versions
```

Comment:
@psihachina can you please why you used "at the same time", and explain the logic? 
Do we still download the binary even if the docker version is used?

Reply:
@AutomationD Please check `update description` commit